server/tapeuse.go

```diff
@@ -0,0 +1,64 @@
+package server
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+// A FixityDB tracks the information the fixity service needs to know what
+
+func (s *RESTServer) EnableTapeUse() {
+	log.Println("Enabling Bendo Tape Use")
+	s.TapeUsed = true
+	s.Items.SetUseStore(true)
+}
+
+func (s *RESTServer) DisableTapeUse() {
+	log.Println("Disabling Bendo Tape Use")
+	s.TapeUsed = false
+	s.Items.SetUseStore(false)
+}
+
+func (s *RESTServer) StatusTapeUse() bool {
+	return s.TapeUsed
+}
+
+// SetTapeUseHandler handles requests to PUT /admin/use_tape/:status
+func (s *RESTServer) SetTapeUseHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	log.Println("Called PUT /admin/user_tape")
+
+	status := ps.ByName("status")
+
+	switch status {
+	case "on":
+		w.WriteHeader(201)
+		// start Enable Process, unless already enabled
+		if s.StatusTapeUse() == false {
+			go s.EnableTapeUse()
+		}
```

Comment:
Don't use `go` here---there is no reason to execute `EnableTapeUse` in a parallel coroutine.

And there is probably no need to use the if statement to guard access since it is just setting a flag.